__mocks__/profiles.ts

```diff
@@ -20,10 +20,11 @@ const TEST_PROFILES = {
     profile1: {
         generationHash: 'ACECD90E7B248E012803228ADB4424F0D966D24149B72E58987D2BF2F2AF03C4',
         profileName: 'profile1',
-        hint: 'password is password',
+        hint: 'password is Password1',
```

Comment:
..lets, uh, not do 'password1'. No, it doesn't need to be secure, but be a bit creative in your naming so you don't overlap with other products. 

You could have chosen rum; oxford; billboard; lotus...any password and name combination!

Reply:
`Password1` is the improved version over old `password` :) picked a simple password to show that it requires a capital and a numeric char.